main.go

```diff
@@ -0,0 +1,11 @@
+package main
+
+import "fmt"
+
+func main() {
+	fmt.Printf("%s")
```

Comment:
__[gometalinter]__
* [vet] missing argument for Printf("%s"): format reads arg 1, have only 0 args